packages/webapp/pages/popup/notifications/subscribe.tsx

```diff
@@ -0,0 +1,92 @@
+import classNames from 'classnames';
```

Comment:
Instead of `subscribe`, can we name it `enable` please?

Reply:
Definitely. Let's do it 🚢